src/main/java/blackjack/domain/card/Value.java

```diff
@@ -0,0 +1,37 @@
+package blackjack.domain.card;
+
+public enum Value {
+    ACE(1, 11),
+    TWO(2, 2),
+    THREE(3, 3),
+    FOUR(4, 4),
+    FIVE(5, 5),
+    SIX(6, 6),
+    SEVEN(7, 7),
+    EIGHT(8, 8),
+    NINE(9, 9),
+    TEN(10, 10),
+    JACK(10, 10),
+    QUEEN(10, 10),
+    KING(10, 10);
+
+    private final int minScore;
+    private final int maxScore;
+
+    Value(int minScore, int maxScore) {
+        this.minScore = minScore;
+        this.maxScore = maxScore;
+    }
+
```

Comment:
```suggestion
    Value(int score) {
        this(score, score);
    }
```
오버로딩을 통해 `minScore`와 `maxScore`가 같은 경우를 두 번 적지 않고 처리할 수 있겠네요!

Reply:
와!!! 안그래도 이번주에 주생성자와 부생성자에 대해 알게 되어서 적용을 해 보고 싶었는데요!!
바로 적용을 해 보았습니다!